### PR TITLE
Forms cookbook

### PR DESCRIPTION
**Pull request by Crista2019**
- Follows points in checklist from issue #7 to create a companion project for the future Forms cookbook tutorial.
- Addresses errors caught by linter in IntelliJ, but likely missing important logic and/or comments to explain code to the reader.

Making a PR now to allow for feedback and comments on specific lines of code.

### CODE REVIEW

File: forms/src/main/java/swim/basic/CustomClient.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      // Commands the addValue lane of UnitAgent with both of the aforementioned messages.
+      swimClient.command(hostUri, fooNodeUri, "addValue", fooVal);
+      swimClient.command(hostUri, barNodeUri, "addValue", barVal);
+
+      Thread.sleep(5000 * i);
+    }
+    System.out.println("Will shut down client");
+    swimClient.stop();
+  }
+}
```
Comment (brohitbrose) [inline forms/src/main/java/swim/basic/CustomClient.java:37]: It's generally a good practice to end source files with a single blank line; it even gets a shoutout in the POSIX standard! https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206

File: forms/src/main/java/swim/basic/FooType.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+			form = Form.forClass(FooType.class);
+		}
+		return form;
+	}
+
+	@Override
+	public String toString() {
+		return Recon.toString(form().mold(this));
+	}
+}
```
Comment (brohitbrose) [inline forms/src/main/java/swim/basic/FooType.java:47]: Add newline, as discussed above.

File: settings.gradle [hunk 3]
```diff
@@ -31,4 +31,7 @@ include 'join-value-lanes'
 project(':join-value-lanes').projectDir = file('join_value_lanes')
 
 include 'join-map-lanes'
-project(':join-map-lanes').projectDir = file('join_map_lanes')
\ No newline at end of file
+project(':join-map-lanes').projectDir = file('join_map_lanes')
+
+include 'forms'
+project(':forms').projectDir = file('forms')
```
Comment (brohitbrose) [inline settings.gradle:37]: Add newline

File: forms/src/main/java/swim/basic/UnitAgent.java [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+  		});
+	
+  // CommandLane<Value> addValue, whose onCommand(value) callback prints value
+  // Just included to exercise that the internal lane type is always fundamentally a Value, so this type is compatible with anything.
+  @SwimLane("addValue")
+  public CommandLane<Value> addValue = this.<Value>commandLane()
+  	.onCommand((Value value) -> {
+  		System.out.println("some value: " + value);
+  	});
+}
```
Comment (brohitbrose) [inline forms/src/main/java/swim/basic/UnitAgent.java:47]: Add newline

File: forms/src/main/java/swim/basic/BasicPlane.java [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+import swim.kernel.Kernel;
+import swim.server.ServerLoader;
+import swim.structure.Value;
+
+public class BasicPlane extends AbstractPlane {
+
+  @SwimRoute("/unit/:id")
+  AgentRoute<UnitAgent> unitAgentType;
+
+  public static void main(String[] args) {
```
Comment (brohitbrose) [inline forms/src/main/java/swim/basic/BasicPlane.java:17]: I like @ajay-gov 's suggestion about first exercising `Form`s without the Swim stack. Will comment on #7 about how to best approach this.

File: forms/src/main/java/swim/basic/BarType.java [hunk 6]
```diff
@@ -0,0 +1,92 @@
+package swim.basic;
+
+import swim.recon.Recon;
+import swim.structure.*;
```
Comment (brohitbrose) [inline forms/src/main/java/swim/basic/BarType.java:4]: For whatever reason, we typically don't use `.*` imports at swim. IntelliJ should let you alt+enter to expand them automatically. Alt+enter is just great in general.

**Comment by Crista2019** [comment]
Am I able to merge this PR, or is there anything to fix/wait on here?

**Comment by brohitbrose** [comment]
Sorry about that, forgot to hit `Approve` after you pushed the new changes. Nothing more to fix, so feel free to merge. Thank you!

**Pull request merged**